### PR TITLE
Blob is now much more resistant to explosions.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Blob parts now have a base explosion block of 1, shields have an explosion block of 3, and the core has an explosion block of 6 and is immune anyway.
Also gives blob types and mobs better descriptions because WOW these were bad.

### CODE REVIEW

**Comment by daman997** [comment]
yes pls :+1:

**Comment by CosmicScientist** [comment]
:+1: Descriptions are a nice touch.

**Comment by SconesC** [comment]
:-1: Only shield blobs should be stopping bombs, encourages intelligent blast-resistant-layer placement instead of mongoloid bomb immunity

File: code/game/gamemodes/blob/blobs/shield.dm [hunk 1]
```diff
@@ -2,8 +2,9 @@
 	name = "strong blob"
 	icon = 'icons/mob/blob.dmi'
 	icon_state = "blob_idle"
-	desc = "Some blob creature thingy"
```
Comment (Miauw62) [inline code/game/gamemodes/blob/blobs/shield.dm]: >removing tgstation culture

**Comment by Kromgar** [comment]
THANK YOU GOD.

I can finally watch a blob round without everything getting ruined by bombs instagibbing the core

File: code/game/gamemodes/blob/blobs/shield.dm (hunk 1 above)
Comment (Anonus) [inline code/game/gamemodes/blob/blobs/shield.dm]: I'm killing the fuck out of it, because I've hated it for as long as I've known it was a thing.

Comment (Thunder12345) [inline code/game/gamemodes/blob/blobs/shield.dm]: Save /tg/station culture 2k15

**Comment by Razharas** [comment]
Cant we just check all the things on the turf to have explosion block value instead of this typechecking fuckery?

**Pull request merged**